source/install/verify-signatures/gpg.txt

```diff
@@ -0,0 +1,90 @@
+.. _compass-verify-signatures-gpg:
+
+==========================================
+Verify Packages with GPG (Linux and macOS)
```

Comment:
In Compass, we can also verify archive (windows) using gpg! Even nupkg. So maybe we should 
```suggestion
Verify Packages with GPG
```

Reply:
Cool! That's good to know.